pkgs/top-level/all-packages.nix

```diff
@@ -36183,17 +36183,37 @@ with pkgs;
 
   clmagma = callPackage ../development/libraries/science/math/clmagma { };
 
-  mathematica = callPackage ../applications/science/math/mathematica { };
+  mathematica = callPackage ../applications/science/math/wolfram {
+    version = "14.0.0";
+  };
+
+  mathematica-webdoc = callPackage ../applications/science/math/wolfram {
+    version = "14.0.0";
+    webdoc = true;
+  };
+
+  mathematica-cuda = callPackage ../applications/science/math/wolfram {
+    version = "14.0.0";
+    cudaSupport = true;
+  };
+
+  mathematica-webdoc-cuda = callPackage ../applications/science/math/wolfram {
+    version = "14.0.0";
+    webdoc = true;
+    cudaSupport = true;
+  };
```

Comment:
I tend to dislike adding these attributes. How about using an alias to point to the new `wolfram` attributes?

Reply:
This is to reserve the name `mathematica` for older versions. Note that the `name` of the derivation, defined in `default.nix`, is dependent on the found `version`. Therefore, the derivations' names will begin with `mathematica` with the above attributes or `wolfram` without them, reflecting the actual rebranding.

Reply:
> This is to reserve the name `mathematica` for older versions. Note that the `name` of the derivation, defined in `default.nix`, is dependent on the found `version`. Therefore, the derivations' names will begin with `mathematica` with the above attributes or `wolfram` without them, reflecting the actual rebranding.

That's good. But since what most people interact with are `nix search` results which show _attribute_ names and `meta.description`s, I'd focus on these details more.

Reply:
I'm in favor of an alias with a warning here, because at least in the near future "Mathematica" is still going to be the more recognizable name, and it seems undesirable for people to accidentally install the old version because they weren't aware of the rebrand.